doc/classes/@GlobalScope.xml

```diff
@@ -911,8 +913,10 @@
 			Middle Mouse Button Mask
 		</constant>
 		<constant name="BUTTON_MASK_XBUTTON1" value="128" enum="ButtonList">
+			Extra Mouse Button 1 Mask
 		</constant>
 		<constant name="BUTTON_MASK_XBUTTON2" value="256" enum="ButtonList">
+			Extra Mouse Button 1 Mask
```

Comment:
`s/1/2/`

Reply:
Thanks!  Always good to have another set of eyes look over things.